lighthouse-cli/chrome-launcher.ts

```diff
@@ -30,6 +30,12 @@ const log = require('../lighthouse-core/lib/log');
 const spawn = childProcess.spawn;
 const execSync = childProcess.execSync;
 
+const green = '\x1B[32m';
+const reset = '\x1B[0m';
+
+// See https://github.com/GoogleChrome/lighthouse/issues/1228
+const tick = process.platform === 'win32' ? '\u221A' : '✓';
```

Comment:
Yeah, unfortunately, the normal tick is bold and looks a lot better on normal terminals, i.e. not Windows :/

I'll add the `const` now and this can be refactored later, see https://github.com/GoogleChrome/lighthouse/pull/1307#issuecomment-269895427